tests/list/assignments.spec.ts

```diff
@@ -0,0 +1,81 @@
+import {VariableCommand} from "../../src/commands/variable.command";
+import axios from "axios";
+import {logger} from "../../src/util/logger";
+import {TestTransport} from "../utls/test-transport";
+import {setDefaultProfile} from "../utls/context-mock";
+import {FileService} from "../../src/services/file-service";
+import * as fs from "fs";
+import * as path from "path";
+
+jest.mock("axios");
```

Comment:
I think these are better suited as part of a global setup instead of here, wdyt?

Reply:
I agree and added a test setup file that handles mocking modules and the default profile.